CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 * [FEATURE] The distributor can now drop labels from samples (similar to the removal of the replica label for HA ingestion) per user via the `distributor.drop-label` flag. #1726
 * [FEATURE] Added `global` ingestion rate limiter strategy. Deprecated `-distributor.limiter-reload-period` flag. #1766
 * [FEATURE] Added support for Microsoft Azure blob storage to be used for storing chunk data. #1913
+* [FEATURE] EXPERIMENTAL: Added user sub rings for use with TSDB backend storage. #1947
```

Comment:
Technically this feature is not related to the TSDB storage (even if that's your use case). I would not mention it here, while I would mention the CLI flag controlling it.

Reply:
Done.